### PR TITLE
Update tensorflow-xla-runtime.spec

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

File: tensorflow-xla-runtime.spec [hunk 1]
```diff
@@ -12,8 +12,13 @@ cp -r ${PY3_TENSORFLOW_ROOT}/lib/python%{cms_python3_major_minor_version}/site-p
 
 export CPATH="${CPATH}:${EIGEN_ROOT}/include/eigen3"
 
+%define cxxflags -fPIC
```
Comment (smuzaffar) [inline tensorflow-xla-runtime.spec]: thanks @iarspider. Can you please include `## INCLUDE compilation_flags` and add `%{?arch_build_flags}` in the `cxxflags` here.

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

**Comment by iarspider** [comment]
please abort

**Comment by smuzaffar** [comment]
@iarspider thanks for reminding me that we should also use the eigen flags here

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

File: tensorflow-xla-runtime.spec [hunk 2]
```diff
@@ -9,11 +12,16 @@ BuildRequires: cmake
...
 export CPATH="${CPATH}:${EIGEN_ROOT}/include/eigen3"
 
+CXXFLAGS="-fPIC %{arch_build_flags} ${CMS_EIGEN_CXX_FLAGS}"
+%ifarch x86_64
+    CXXFLAGS="${CXXFLAGS} -msse3"
+%endif
+
 pushd tensorflow/xla_aot_runtime_src
-  cmake . -DCMAKE_CXX_FLAGS="-fPIC -msse3" -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} -DBUILD_SHARED_LIBS=OFF
+  cmake . -DCMAKE_CXX_FLAGS=${CXXFLAGS} -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} -DBUILD_SHARED_LIBS=OFF
```
Comment (smuzaffar) [inline tensorflow-xla-runtime.spec]: please use `-DCMAKE_CXX_FLAGS="${CXXFLAGS}"`

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

**Comment by smuzaffar** [comment]
please test for el8_ppc64le_gcc12

**Comment by smuzaffar** [comment]
FYI @iarspider https://github.com/tensorflow/tensorflow/issues/58481

File: tensorflow-xla-runtime.spec [hunk 3]
```diff
@@ -9,11 +12,16 @@ BuildRequires: cmake
...
 
 %build
-
+source %{_sourcedir}/env
 export CPATH="${CPATH}:${EIGEN_ROOT}/include/eigen3"
 
+CXXFLAGS="-fPIC %{arch_build_flags} ${CMS_EIGEN_CXX_FLAGS}"
+%ifarch x86_64
+    CXXFLAGS="${CXXFLAGS} -msse3"
+%endif
```
Comment (smuzaffar) [inline tensorflow-xla-runtime.spec:21]: I guess adding `-DEIGEN_NEON_GEBP_NR=4` for aarch64 should fix this

Comment (smuzaffar) [reply tensorflow-xla-runtime.spec:21]: or better to update scram-tools.file/tools/eigen/env.file and add this flagfor aarch64 there

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

**Comment by iarspider** [comment]
please test for el8_aarch64_gcc12

**Comment by iarspider** [comment]
please test for el8_ppc64le_gcc12

**Comment by smuzaffar** [comment]
please test

**Comment by iarspider** [comment]
> `---> test testPhysicsToolsSelectorUtilsPythonTestsDriver had ERRORS`
Error occurs in IBs, not related to this PR.

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**